library/protoc-gen-validate-cpp/v0.6.12/buf.plugin.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+name: buf.build/library/protoc-gen-validate-cpp
```

Comment:
I don't think we need to call this `protoc-gen-validate-cpp`, maybe `validate-cpp` is enough?